src/main/features/claim/routes/pay.ts

```diff
@@ -102,8 +74,30 @@ async function successHandler (res, next) {
       features += features === undefined ? 'directionsQuestionnaire' : ', directionsQuestionnaire'
     }
     await claimStoreClient.saveClaim(draft, user, features)
-  }
 
+  } catch (err) {
+    if (err.statusCode === HttpStatus.INTERNAL_SERVER_ERROR || err.statusCode === HttpStatus.SERVICE_UNAVAILABLE) {
+      logError(
+        user.id,
+        draft.document.claimant.payment,
+        `Payment processed successfully but there is problem saving claim from claim store externalId: ${externalId}.`
+      )
+      trackCustomEvent(`Post payment successful but no claim in claim-store for externalId: ${externalId}`,
+        {
+          externalId: externalId,
+          payment: draft.document.claimant.payment,
+          error: err
+        })
+      next(err)
+      return
+    } else if (err.statusCode === HttpStatus.CONFLICT) {
+      logError(
+        user.id,
+        draft.document.claimant.payment,
+        `Payment processed successfully and claim already exist with same externalId: ${externalId}.`
```

Comment:
Can we fix the grammar please?
`... and claim '${externalId}' already exists.`

Reply:
done